### PR TITLE
Don't index kpt.dev while in development

### PR DESCRIPTION
**Pull request by etefera**
Fixes https://github.com/GoogleContainerTools/kpt/issues/1536
Adds KPT to page title.

### CODE REVIEW

File: site/index.html [hunk 1]
```diff
@@ -2,7 +2,8 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <title>Home</title>
+    <meta name="robots" content="noindex" />
```
Comment (droot) [inline site/index.html:5]: Is it enough ? or we need `robots.txt` at the root ?

Comment (etefera) [reply site/index.html:5]: This is the only HTML served from the domain so any page would be crawled with that attribute https://developers.google.com/search/docs/advanced/robots/robots_meta_tag

Comment (droot) [reply site/index.html:5]: ok.

**Review by droot — APPROVED** [review]

File: site/index.html [hunk 2]
```diff
@@ -2,7 +2,8 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <title>Home</title>
+    <meta name="robots" content="noindex" />
+    <title>KPT - Home</title>
```
Comment (frankfarzan) [inline site/index.html:6]: we should use small cap `kpt` consistently everywhere.

**Review by frankfarzan — COMMENTED** [review]
lgtm after fix

**Review by frankfarzan — APPROVED** [review]

File: site/index.html (hunk 2 above)
Comment (etefera) [reply site/index.html]: SGTM, done.

**Pull request merged**